netaddr.go

```diff
@@ -278,12 +280,14 @@ func (ip IP) v6(i uint8) uint8 {
 // The zero value is not a valid IP address of any type.
 //
 // Note that "0.0.0.0" and "::" are not the zero value.
+//gcassert:inline
```

Comment:
We don't want this in the godoc. Or do all the tools strip these?

Reply:
I had the same concern, but it seems like `go doc` at least removes it:
```
✔ ~/src/inet.af/netaddr [mdl-gcassert|⚑ 1] 
17:16 $ go doc IP.IsZero
package netaddr // import "inet.af/netaddr"

func (ip IP) IsZero() bool
    IsZero reports whether ip is the zero value of the IP type. The zero value
    is not a valid IP address of any type.

    Note that "0.0.0.0" and "::" are not the zero value.

```

Reply:
Is that done at a low layer so godoc.org etc also gets it?

Reply:
I can confirm that `golang.org/x/tools/cmd/godoc` ignores the gcassert annotations, so presumably godoc.org and pkg.go.dev would as well.